### PR TITLE
add "File Extensions" section with text on .njk adopted extension.

### PR DESCRIPTION
**Pull request by dkebler**
For consistency capitalize the handfull of lower case "nunjucks" found in the text.

### CODE REVIEW

File: docs/templating.md [hunk 1]
```diff
@@ -14,6 +14,10 @@ This is an overview of the templating features available in nunjucks.
 > lacking here. Read about the differences
 > [here](http://mozilla.github.io/nunjucks/faq.html#can-i-use-the-same-templates-between-nunjucks-and-jinja2-what-are-the-differences).
 
+## File Extensions
+
+Although you are free to use any file extension you wish for your Nunjucks template files the Nunjucks community has adopted  `.njk`.  If you are developing tools or editor syntax helpers for Nunjucks please include recognition of this extension.
```
Comment (ricordisamoa) [inline docs/templating.md]: Reference?

Comment (carljm) [inline docs/templating.md]: It was discussed do the mailing list. I said I thought it made sense to recommend something, and I didn't care between .nj, .njs, or .njk.

Comment (dkebler) [inline docs/templating.md]: https://groups.google.com/forum/#!topic/nunjucks/mJbJ96OPUGQ

Comment (carljm) [inline docs/templating.md]: Should be a comma after "Nunjucks template files"

**Comment by carljm** [comment]
The docs are hard-wrapped at 70 columns; please update this PR to do the same. Made one grammatical comment; otherwise this looks fine, thanks! (Please comment when you've made that update, GH doesn't notify when a PR's code is updated).

**Comment by dkebler** [comment]
just getting back to this.  guess my branch is out of date to master.  Can still merge?

**Comment by dkebler** [comment]
closing this PR in favor of PR #691

**Pull request closed without merging**